python/tests/kernel/mock/hello/dtype.py

```diff
@@ -0,0 +1,14 @@
+# ============================================================================ #
+# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
+# All rights reserved.                                                         #
+#                                                                              #
+# This source code and the accompanying materials are made available under     #
+# the terms of the Apache License 2.0 which accompanies this distribution.     #
+# ============================================================================ #
+import cudaq 
+from dataclasses import dataclass
+
+@dataclass
+class TestClass:
+    i : int 
+    d : float 
```

Comment:
Q: Only `int` and `float` members are supported, right? What about boolean? (If already supported, perhaps add one test to cover that)

Reply:
Right now I have int, float, bool, and list supported.